README.md

```diff
@@ -1,110 +1,10 @@
-# Cloud Foundry Terraform Provider [![Build Status](https://travis-ci.org/cloudfoundry-community/terraform-provider-cloudfoundry.svg?branch=master)](https://travis-ci.org/cloudfoundry-community/terraform-provider-cloudfoundry)
+# Cloud Foundry Terraform Provider (v3 API)
 
+Experimental implementation of terraform resource for rolling deployments of
+cloudfoundry applications using the v3 API.
 
-Overview
---------
+Long-term intension is to contribute back
```

Comment:
s/intension/intention/

Reply:
stupid non-phonetic language... fixed

Reply:
It'll be fine when we all switch to korean